Gems/ROS2/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
This file can be removed as it is covered in the root file https://github.com/o3de/o3de-extras/blob/development/LICENSE_APACHE2.TXT

Reply:
Done